release/_publishing.sh

```diff
@@ -1,5 +1,47 @@
 #!/bin/bash
 
+function add_fixed_issues_to_project_version {
+	IFS=',' read -r -a fixed_issues_array < "${_BUILD_DIR}/release/release-notes.txt"
+
+	local fixed_issues_array_length="${#fixed_issues_array[@]}"
+
+	local fixed_issues_array_part_length=$((fixed_issues_array_length / 4))
+
+	local patcherProjectVersionId="${1}"
```

Comment:
@kiwm, can you move this declaration to the first line? Let's try to use the first lines of the function to receive the parameters sent to it. :wink: